Source/WebCore/html/parser/HTMLSrcsetParser.cpp

```diff
@@ -56,7 +56,7 @@ template<typename CharType>
 static void appendDescriptorAndReset(const CharType*& descriptorStart, const CharType* position, Vector<StringView>& descriptors)
 {
     if (position > descriptorStart)
-        descriptors.append(StringView(descriptorStart, position - descriptorStart));
+        descriptors.append(StringView(std::span(descriptorStart, position - descriptorStart)));
```

Comment:
I don’t think we need the explicit `StringView` here. Probably many, **many** other cases like this.